assets/components/src/modal/style.scss

```
@@ -16,7 +16,7 @@
 	width: 100%;
```

Adding `max-height: 100%` ( or `100vh`) lets the modal content scroll if it exceeds the height of the viewport:

```suggestion
	width: 100%;
	max-height: 100%;
```

---

Fixed in https://github.com/Automattic/newspack-plugin/pull/1013/commits/a543fba520a2d7eaa83d76145f4de75a00e3dd5a